examples/__testfixtures__/rename-onclick/types.input.tsx

```diff
@@ -0,0 +1,13 @@
+import { Pressable } from 'react-native';
+
+type Props = {
+    onClick: () => void;
```

Comment:
Should we remove imported `onClick` types too? Or maybe just do that manualy? 

There are two, that I'm seeing
```
import {  OnClick } from '@rbilabs/components-library';
```
and
```
import { OnClick } from '@rbi-ctg/frontend'
```